config/initializers/paperclip.rb

```diff
@@ -0,0 +1,3 @@
+Paperclip::Attachment.default_options[:path] = '/:class/:attachment/:id_partition/:style/:filename'
+Paperclip::Attachment.default_options[:s3_host_name] = 's3-eu-central-1.amazonaws.com'
```

Comment:
Line is too long. [86/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.